cmd/webhook/main.go

```diff
@@ -19,15 +19,23 @@ package main
 import (
 	"os"
 
+	"github.com/go-logr/logr"
 	"k8s.io/component-base/cli"
 	_ "k8s.io/component-base/logs/json/register" // for JSON log format registration
 	controllerruntime "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log"
 
 	"github.com/karmada-io/karmada/cmd/webhook/app"
 )
 
 func main() {
 	ctx := controllerruntime.SetupSignalHandler()
+	// Controller-runtime expects its consumers to set a logger through log.SetLogger.
```

Comment:
```suggestion
	// After controller-runtime 0.15.0, Controller-runtime expects its consumers to set a logger through log.SetLogger.
```

Woud you want to add the version info here?